components/scream/src/share/tests/common_physics_functions_tests.cpp

```diff
@@ -20,7 +20,7 @@ struct UnitWrap::UnitTest<D>::TestUniversal
   static void run()
   {
     using physicscommon      = scream::PhysicsFunctions<Device>;
-    using physicscommon_host = scream::PhysicsFunctions<HostDevice>;
+    using physicscommon = scream::PhysicsFunctions<HostDevice>;
```

Comment:
This line must be removed. You can't redefine `physicscommon` to be something else.